docs/source/existing_code.rst

```diff
@@ -0,0 +1,174 @@
+.. _existing-code:
+
+Using mypy with an existing codebase
+====================================
+
+This section explains how to get started using with an existing,
+significant codebase that has no or few type annotations. If you are
+a beginner, you can skip this section.
+
+These steps will get you started with mypy on an existing codebase:
+
+1. Start small -- get a clean mypy build for some files, with few
+   annotations
+
+2. Write a mypy runner script to ensure consistent results
+
+3. Run mypy in Continuous Integration to prevent type errors
+
+4. Gradually annotate commonly imported library modules
+
+5. Write annotations as you modify existing code and write new code
+
+6. Use MonkeyType or PyAnnotate to automatically annotate legacy code
+
+We discuss all of these points in some detail below, and a few optional
+follow-up steps.
+
+Start small
+-----------
+
+If your codebase is large, pick a subset of your codebase (say, 5,000
+to 50,000 lines) and run mypy only on this subset at first,
+*without any annotations*. This shouldn't take more than a day or two
+to implement, so you start enjoying benefits soon.
+
+You'll likely need to fix some mypy errors, such as by inserting
+annotations requested by mypy, or by adding ``# type: ignore``
```

Comment:
"such as by" -> "either by"

I think we can also get rid of the comma before the "or".